projects/managed-hubs/support/process.md

```diff
@@ -153,6 +154,29 @@ When a new ticket lands in Freshdesk under the support group and it is not an in
 
     **Close the request**. If the Community Representative confirms that their request has been fulfilled, consider this request closed. In FreshDesk, mark the incident as {guilabel}`Closed`.
 
+### End of rotation hand-off
+
+Below is the process describing the expectations before a triager's end of rotation.
+
+1. Processed ~all unassigned tickets
+
+_Support triagers should aim to process each unassigned ticket before the end of the rotation._
+
+This helps us ensure no tickets becomes stale and unattended, and nudges us to get tickets closed or resolved rather than stuck open or pending indefinitively. Tickets left open becomes weight over time.
+
+2. Unassign from tickets
+
+_Support triagers should by end of rotation unassign themselves as agents from all tickets._
+
+This helps making responsibilities unambigious. A ticket's assigned agent should be either nobody or one of the current triagers.
+
+3. Write handover notes
+
+_Support triagers should, when they unassign from a ticket ensure a handover note is available and up to date towards the end of the ticket._
+
+A ticket handover note should _summarize whats relevant_ to know in order to drive the ticket towards a resolution without needing to read through the conversation up to that point.
```

Comment:
@sgibson91 wrote in https://github.com/2i2c-org/meta/issues/944#issuecomment-2061625516:

> Also maybe some guidance on what good handover notes look like? I am intimidated by this piece, usually because I am struggling to process the ticket in the first place.

I've tried to write a few handover notes, but its tough for me to say if they are good as being good/bad is to a large extent determined if the reader of the handover note. I think after we've all attempted to write and read a few it will be easier to say what are good examples.

---

For practical quick reference though, I looked at some open / pending tickets and found notes I've written so far:

> There is no action point besides handling the opened tickets at this point.

> Marked as pending awaiting resolution of https://github.com/2i2c-org/docs/issues/224 tracking writing dask-gateway usage documentation allowing communities with dask-gateway hubs actually make use of it.

I also recall writing lengthy notes for tickets that are wild with many separate things discussed - then I remember outlining the individual parts and marking them as completed or clarifying what remains.

Reply:
@sgibson91 are you okay with going for the PR as it is now and trying to collectively learn what a good handover note is?